src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -107,17 +118,23 @@ const NavHeader = (props) => {
 
   const heightWithCoverImage = () => {
     if (coverPath) {
-      return {height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER + topPosition};
+      return {height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER};
     }
     return null;
   };
 
+  const AnimatedBlurView = Animated.createAnimatedComponent(BlurView);
+
   return (
-    <Animated.View style={styles.container(animatedHeight)}>
+    <View style={{}}>
       <StatusBar barStyle="dark-content" />
       <View
-        source={{uri: coverPath}}
-        style={[styles.navContainer(isHeaderHide, topPosition), heightWithCoverImage()]}
+        style={[
+          styles.navContainer(isHeaderHide),
+          {
+            height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+          }
+        ]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `topPosition` from the height calculation might cause issues with the layout on iOS devices with a notch. Please verify that the layout still looks correct on these devices after this change.
```diff
- return {height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER};
+ return {height: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER + topPosition};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
we want to remove that for sure, design it's bit changed